frontend/src/components/UserForm/UserForm.js

```diff
@@ -0,0 +1,198 @@
+import { useMutate } from "../../hooks/use-mutate"
+import { useRef, useState, useEffect} from "react"
+import { useUser } from "../../hooks/user-hooks"
+import { useParams } from "react-router"
+import { CountryDropdown, RegionDropdown } from "react-country-region-selector";
+
+const UserForm = () => {
+  const formRef = useRef()
+  const { updateUser } = useMutate()
+  const { id } = useParams()
+  const user = useUser({ "id": id })
+  const isAdmin = user?.data?.data?.data.user.admin
+
+  const fetchedCountry = user?.data?.data?.data?.user.country;
+  const fetchedState = user?.data?.data?.data?.user.state;
+  const [country, setCountry] = useState("");
+  const [region, setRegion] = useState("");
+
+  useEffect(() => {
+    if (fetchedCountry) setCountry(fetchedCountry);
+    if (fetchedState) setRegion(fetchedState);
+  }, [fetchedCountry, fetchedState]);
+
+  if (user.isError) return <h1>Something went wrong!</h1>
+  if (user.isLoading) return <h1>Loading...</h1>
+
+  const handleSubmit = async (e) => {
+    e.preventDefault()
+    // Add the current country/state values to the form data
+    formRef.current.append("country", country);
+    formRef.current.append("state", region);
```

Comment:
What is this doing?
This looks to be appending the country and region text to the bottom of the form whenever `handleSubmit` is called. The input already has entries for the country and state. This is causing additional sections to be added to the form upon submission.
<img width="574" alt="image" src="https://user-images.githubusercontent.com/5695484/227595997-99a4ffe7-0306-4f92-ae82-45ec64896dd1.png">

Reply:
forgot to remove that, removed